pkg/webhooks/leaderworkerset_webhook_test.go

```diff
@@ -184,3 +189,100 @@ func TestIsNotMoreThan100Percent(t *testing.T) {
 		})
 	}
 }
+
+func TestGeneralValidate(t *testing.T) {
+	type args struct {
+		obj runtime.Object
+	}
+	tests := []struct {
+		name    string
+		r       *LeaderWorkerSetWebhook
```

Comment:
The `r` seems not used in the following code. :-)

Reply:
done